src/Assignments/ProblemIframe.tsx

```diff
@@ -82,50 +163,33 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
             // console.error('Hijacker: Could not find the form to insert the listener');
             return;
         }
-        problemForm.addEventListener('submit', (event: { preventDefault: () => void; }) => {
+
+        problemForm.addEventListener('submit', _.debounce((event: { preventDefault: () => void; }) => {
```

Comment:
I thought we discussed moving the debounce to prepare and submit in this case since other buttons are considered submits (i.e. geogebra reset)

Reply:
We want different denounce behavior for save vs submit. Submit is leading edge and save is trailing.